src/core/system.js

```diff
@@ -310,19 +311,21 @@ export default class Store {
 
 }
 
-function combinePlugins(plugins, toolbox) {
+function combinePlugins(plugins, toolbox, pluginLoadType) {
```

Comment:
Ok, revised comment. we should receive the `pluginOptions` object here

Reply:
where we pass in the `pluginsOption` in line 67 above

Reply:
I also think we should be able to get `pluginsOptions` via `this.getConfigs().pluginsOptions`

Reply:
> I also think we should be able to get `pluginsOptions` via `this.getConfigs().pluginsOptions`

It seems to be handled by this part : https://github.com/swagger-api/swagger-ui/blob/master/src/core/index.js#L163
Can you confirm ?

Reply:
re: this.getConfigs(), I was thinking of something else. sorry for the confusion.